.github/workflows/check-package-managers.yml

```diff
@@ -0,0 +1,27 @@
+name: Check package managers
```

Comment:
I think we can update the names of our workflows and jobs so they read a little better in the bottom of the PR.

Maybe something like ...
```
Tests / Unit
Tests / UI
Tests / Integration
Build Demo / Carthage
Build Demo / CocoaPods
```

Reply:
I'm down for renaming the jobs! But the Carthage check doesn't actually build a demo app. It's just running `carthage build --no-skip-current`, which I think just makes sure the frameworks build.

Reply:
Maybe we could do:
```
Tests / Unit
Tests / UI
Tests / Integration
Build / Carthage
Build / CocoaPods
```

Reply:
I like it 👍

Reply:
Updated!